.github/workflows/contracts.yml

```diff
@@ -3,6 +3,8 @@ name: Solidity
 on:
   push:
     branches:
+      # TODO: Run only on master after we're fully migrated from Circle CI
+      - "rfc-18/**"
       - master
```

Comment:
Can we remove `master` for now? In client workflow we're running it only on `rfc-18/**` branches. We should probably do the same here.

Reply:
In case of client workflow we have equivalent of it present in CircleCI, executing for master, so there was no point in executing it for macter in GH Actions. 
In case of contracts workflow, there is no equivalent of this workflow in CircleCI, hence executing it for events on master is not duplicating anything.